schemas/arb.json

```diff
@@ -6,12 +6,292 @@
     "type": "object",
     "patternProperties": {
         "^@@x-.+$": {
-            "type": "string",
+            "type": [
+                "string",
+                "number",
+                "integer",
+                "boolean"
+            ],
             "description": "A custom global attribute."
         },
         "^@(?!@).+$": {
-            "type": "string",
-            "description": "Metadata for a message"
+            "type": "object",
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "enum": [
+                        "text",
+                        "image",
+                        "css"
+                    ],
+                    "description": "Describes the type of resource. Possible values are \"text\", \"image\", or \"css\". Program should not rely on this attribute in run time. It is mainly for the localization tools."
```

Comment:
nit: for consistency, I'd remove the word "Describes" here (and "It describes (in text)" below). The other descriptions are written without this (for example "A named placeholder", "The type of the parameter", etc.)